p2panda-js/scripts/compile-wasm.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
```

Comment:
I think this is the most usable non-trivial shell script I have ever seen 👏

Reply:
wow, nice script!